components/brave_rewards/resources/page/components/pending_contributions_modal.style.ts

```diff
@@ -0,0 +1,170 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import styled from 'styled-components'
+
+import * as mixins from '../../shared/lib/css_mixins'
+
+export const root = styled.div`
+  width: calc(100vw - 22px);
+  max-width: 450px;
+  min-height: 450px;
+  max-height: calc(100vh - 40px);
+  overflow: auto;
+  border-radius: 8px;
+  background: var(--brave-palette-white);
+`
+
+export const header = styled.div`
+  display: grid;
+  grid-template-columns: 1fr 18px;
+  grid-template-rows: 1;
+  align-items: center;
+  border-bottom: 1px solid rgba(0, 0, 0, 0.12);
+  padding: 21px 24px;
+
+  button {
+    grid-column-start: 2;
+    justify-self: end;
+  }
+
+  div {
+    display: flex;
+    align-items: baseline;
+    padding-top: 2px;
+  }
+`
+
+export const title = styled.span`
+  grid-column-start: 1;
+  justify-self: start;
+  line-height: 22px;
+  font-size: 20px;
+  font-weight: 500;
+  font-color: rgba(33, 37, 41, 1);
+`
+
+export const constributionList = styled.div`
+  height: calc(100% - 16px - 67px);
+  overflow-y: scroll;
+`
+
+/* Each row is a grid
+    ---------------------------------------------
+    |   | 1(29px) | 2(1fr) |  3(1fr)  | 4(14px) |
+    ---------------------------------------------
+    | 1 |  Icon     Site              |     |_| |
+    | 2 |         | Pending Amount              |
+    | 3 |         | Type  |                Date |
+    ---------------------------------------------
+*/
```

Comment:
👍🏻

Reply:
Copied from @mkarolin 's excellent work on the Android page!